frontend/src/app/event/event-page/event-page.component.ts

```diff
@@ -91,7 +180,26 @@ export class EventPageComponent implements OnInit {
    * @param query: Search bar query to filter the items
    */
   onSearchBarQueryChange(query: string) {
-    this.eventsPerDay = this.eventService.groupEventsByDate(this.events, query);
+    this.query = query;
+    let paginationParams = this.page.params;
+    paginationParams.ascending = 'true';
+    if (query == '') {
+      console.log('hi');
+      paginationParams.range_start = this.startDate.toLocaleString('en-GB');
+      paginationParams.range_end = this.endDate.toLocaleString('en-GB');
```

Comment:
This should really be en-US, yeah? Or is there something I'm missing about great britain representation.

Reply:
I'm going to try pushing this fix, it doesn't seem to be breaking anything on my end, but could use a sanity check on your end if there are any cases I'm not thinking of.